docs/file-reference.md

```diff
@@ -869,6 +884,61 @@ secrets:
 Anything [EnvVarSource Spec](https://kubernetes.io/docs/api-reference/v1.8/#envvarsource-v1-core) from Kubernetes can be included within the Kedge file.
 
 
+## imageStream
+
+```yaml
+imageStreams:
+- <imageStream>
+```
+
+> Example
+
+```yaml
+name: rubyapp
+imageStreams:
+- name: rubystream
+  dockerImageRepository: "docker.io/openshift/ruby-20-centos7"
+```
+
+<aside class="notice">
+Each "imageStream" is an OpenShift <a target="_blank" href="https://docs.openshift.org/latest/rest_api/apis-image.openshift.io/v1.ImageStream.html#object-schema">ImageStream Spec</a> with additional Kedge-specific keys.
```

Comment:
"imageStreamObject is an Opendhift ....."

Reply:
fixed